### DISCUSSION TITLE
Doesn't support nodejs < 14?

### DISCUSSION THREAD

**Discussion by jqhr** [pkgxdev/pkgx#799] [Help] [answered]
Command:
```bash
pkgx node@12
```
![图片](https://github.com/pkgxdev/pkgx/assets/23305067/174029da-d7d8-4624-ba31-9cc0ea2f06d0)

For node 14 or above, it works.

**Comment by mxcl** [comment]
Per our FAQ, we can build what you want if you open a ticket. @jhheider did we try to do node 12 already once?

**Reply by jhheider** [reply]
I believe we tried a run once, but didn't dive in on the failures.

Per [endoflife.date](https://endoflife.date/nodejs), node12 hasn't even had security fixes 17 months, so we didn't try too hard. 16 and 14 are both EOL now too.

Most likely aarch64 support was poor, if I'm guessing.

**Reply by mxcl** [reply]
like python 2, people have need of node 12, so we should try.

If aarch64 is a no-go then we may need to figure out how to specify that information in the package.yaml :/

**Reply by jhheider** [reply]
ah, node12 requires python2.7 to build, which we didn't have until recently.

however, there's an issue with drifting version requirements that we're butting up against. i strongly doubt node20 builds with python2.7. i may have to use some tricks. bad tricks.

**Reply by jhheider** [reply]
the trick is ignoring configure, ironically.

[this PR](https://github.com/pkgxdev/pantry/pull/3620) likely will do it.

**Reply by jhheider** [reply, marked as answer]
```shell
$ pkgx --sync
$ pkgx +node^12 node --version
 ✓  ~/.pkgx/npmjs.com/v10.2.0
 ✓  ~/.pkgx/nodejs.org/v12.22.12
v12.22.12
```

**Reply by mxcl** [reply]
beautiful work

**Comment by jqhr** [comment]
Great!
Thanks very very much
